javatests/arcs/core/entity/HandleManagerTestBase.kt

```diff
@@ -349,11 +353,17 @@ open class HandleManagerTestBase {
         val handle = writeHandleManager.createSingletonHandle()
         val handleB = readHandleManager.createSingletonHandle()
         val handleBUpdated = handleB.onUpdateDeferred()
-        handle.store(entity1)
+
+        val expectedCreateTime = System.currentTimeMillis()
+        fakeTime.millis = expectedCreateTime
```

Comment:
does it need to be the real time or can you just do val expectedCreateTime = 123 (I think it makes more obvious that fakeTime is driving the creation time